src/asymmetric_crypto_ctx.cpp

```diff
@@ -463,21 +463,97 @@ namespace mococrw {
         return signDigest(createHash(_impl->hashFunction, message));
     }
 
+namespace {
```

Comment:
Commit message: `s/so that t now/so that it now/`, `s/sanetization/sanitization/`.